pkgs/development/python-modules/manageprojects/default.nix

```diff
@@ -0,0 +1,119 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  cli-base-utilities,
+  cookiecutter,
+  tomlkit,
+  editorconfig,
+  darker,
+  flynt,
+  isort,
+  pygments,
+  autoflake,
+  pyflakes,
+  autopep8,
+  flake8,
+  flake8-bugbear,
+  pyupgrade,
+  refurb,
+  codespell,
+  mypy,
+  click,
+  rich-click,
+  rich,
+  pytestCheckHook,
+  python-gnupg,
+  bx-py-utils,
+  urllib3,
+  pip-tools,
+  typeguard,
+  pkgs,
+}:
+buildPythonPackage rec {
+  pname = "manageprojects";
+  version = "0.19.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jedie";
+    repo = "manageprojects";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ghNx1QngHr9ZV1Y+lJpw6DLpfiUBF/2zRif84qcktCQ=";
+  };
+
+  build-system = [
+    setuptools-scm
+  ];
+
+  dependencies = [
+    cookiecutter
+    tomlkit
+    editorconfig
+    darker
+    flynt
+    isort
+    pygments
+    autoflake
+    pyflakes
+    autopep8
+    flake8
+    flake8-bugbear
+    pyupgrade
+    refurb
+    codespell
+    mypy
+    (cli-base-utilities.overridePythonAttrs { doCheck = false; })
```

Comment:
I'm assuming this is overridden for the same reason as manage-django-project. It should have the same comment

Reply:
It's overridden to prevent a dependency loop since these packages depend on each other :woozy_face: will add a comment